docs/advisors/checks/configuration-pg-check-fsync-enabled.md

```diff
@@ -1,33 +1,40 @@
 # fsync is disabled on PostgreSQL configuration
 
 ## Description
-When this parameter is enabled (which it is by default), the PostgreSQL server tries to make sures that updates are physically written to disk, by issuing fsync() system calls or various equivalent methods mentioned in wal_sync_method. 
+By default, `fsync` is enabled in PostgreSQL so that the PostgreSQL server tries to make sure that updates are physically written to disk by issuing `fsync()` system calls or various equivalent methods mentioned in `wal_sync_method`. 
```

Comment:
Wouldn't be better to simplify with something like:

_Fsync` is enabled by default in PostgreSQL_ ...